modules/canbus/vehicle/ge3/protocol/BUILD

```diff
@@ -0,0 +1,242 @@
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "canbus_ge3_protocol",
+    srcs = [
```

Comment:
When making cc_library, please use one cc file at a time.

Reply:
> What do you really mean? And how I should to modify it?

Please only use one .cc file, for example:
cc_libraray(
      name = "canbus_ge3_protocol_pc_bcm_201",
      srcs = [ "pc_bcm_201.cc"],
      hdrs = [ "pc_bcm_201.h"],
....
)